docs/configuration/http_filters/fault_filter.rst

```diff
@@ -66,6 +67,12 @@ upstream_cluster:
   the config are present in the request with the same values (or based on presence if the ``value``
   field is not in the config).
 
+downstream_nodes:
+  *(optional, array)* Faults are injected for the specified list of downstream hosts. If this setting is
+  not set, faults are injected for all downstream nodes. Downstream node name is taken from
+  :ref:`the HTTP x-envoy-downstream-service-node <config_http_conn_man_headers_downstream-service-node>`
+  header and compared against downstream_nodes list.
+
```

Comment:
I do not see how the schema above when we have two filters with different nodes in the "x-envoy-downstream-service-node" allows us to switch to the mode where we do not care about downstream node at all.

For example, I'm running a test where I inject failures on "canary" only node and now I want to switch to the mode where I do not care about downstream-node at all. How would I do that in the current header matching logic?